extensions/default/src/utils/getDirectURL.js

```diff
@@ -29,31 +29,31 @@ const getDirectURL = (config, params) => {
   }
 
   const value = instance[tag];
-  if (!value) {
```

Comment:
I don't understand what value there is in increasing the indent level.  The recommended approach for handling things is to return quickly at the top of the function, rather than nesting a function inside, as was done originally.

Reply:
Ok, I see what you are doing, github made it non obvious that there was more code - cut things off below.  What I think should be done is to separate this out into two functions, one of which constructs a URL, the other of which uses the value - and export both of them so that they can be used independently, or combined.  Then getDirectURL becomes:

```
const value = instance[tag];
return getBulkdataValue(value) || createRenderedRetrieve(instance)

Then, you can leave the return immediate as is inside getBulkdataValue.
Also, you might add a note that it is incorrect to not have a value here, or rather it means that the value itself is supposed to be empty, but that some DICOMweb systems incorrectly return empty tag values.
Then, we should handle both:

TAG: null
and
TAG: {}

I'd like to see a unit test to check a couple of the various cases, since I think this logic is getting complex enough to warrant that.  

Sorry for the extra work, but by the time things get this complex, splitting it up makes it easier to figure out what one should expect to receive back.